app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Looks like some views can be removed at all, e.g. User model doesn't include `:confirmable, :lockable` modules, but there are devise views related to these modules added to that PR